source/reference/minio-server/minio-server.rst

```diff
@@ -445,6 +450,19 @@ MinIO Console:
    the external identity provider has a reachable URL to which to send the
    authentication response.
 
+.. envvar:: MINIO_SERVER_URL
+
+   *Optional*
+
+   Specify the URL hostname the MinIO Console should use for connecting to the
```

Comment:
Perhaps `to connect`?

Is this an arbitrary URL, or does it need to be just the domain name? `URL hostname` sounds like you can't include a path.

Also should join the broken sentences in this section. (I'm guessing this text came from elsewhere.)

Reply:
FQDN now that I think about it.

Reply:
Also @harshavardhana @kannappanr i feel like we improved this behavior sufficiently in a recent release, but I can't find the PR.